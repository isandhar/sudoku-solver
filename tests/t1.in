B*78*5E*3**AD*C0
**4**7***C*FA**2
A**********437**
**5***9F*******8
*4**B8***E*793**
**E37C****FDB**4
9F*7**5D*3****8*
5**D*F3*24A8C*0*
*8******B****0D5
**D******8**F*E*
**A*9*F**67***BC
***C*AB****E724*
7A*9*B1***5**63*
D*CEF*7*A****8**
****E*A**D**5***
*63509C**B**E***
