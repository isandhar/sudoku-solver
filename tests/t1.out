B97815E4326ADFC0
0E4137D68C9FAB52
ADF6C28B0514379E
3C52A09FD7EB1468
C46AB8215E0793FD
82E37C0A69FDB514
9F074E5DC3B12A86
51BD6F3924A8CE07
E89F2147BAC360D5
47DB536C1820F9EA
23A09DFE467581BC
651C8AB09FDE7243
7A89DB12E05C463F
DBCEF475A1360829
1024E6A3FD895C7B
F63509C87B42EDA1
